pkgs/development/tools/prototool/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "prototool";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "uber";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0m6g5p3jyf0ja5j9rqmw8mhipdqxk7rcxm0plmh65bxk1n22fzjc";
+  };
+
+  modSha256 = "0l4nqb1c1lqqk70l9qwib1azavxlwghi2fv5siwrr5zw4jysz5ml";
+
+  subPackages = [ "cmd/prototool" ];
```

Comment:
I think it would make sense to a dependency on protoc and wrap prototool and set PROTOTOOL_PROTOC_BIN_PATH and PROTOTOOL_PROTOC_WKT_PATH so that prototool uses the version of protoc in this repo.

Reply:
makes sense, I've updated the expression. Thanks!